Makefile

```diff
@@ -106,19 +106,26 @@ endif
 # Refer to the README for cuDNN install instructions
 ifeq ($(USE_CUDNN), 1)
   ifeq ($(SHELL_UNAME), Linux)
-    ifeq ($(shell [ -d $(HOME)/cudnn-frontend/include ] && echo "exists"), exists)
-      $(info ✓ cuDNN found, will run with flash-attention)
-      CUDNN_FRONTEND_PATH ?= $(HOME)/cudnn-frontend/include
-    else ifeq ($(shell [ -d cudnn-frontend/include ] && echo "exists"), exists)
-      $(info ✓ cuDNN found, will run with flash-attention)
-      CUDNN_FRONTEND_PATH ?= cudnn-frontend/include
-    else
-      $(error ✗ cuDNN not found. See the README for install instructions and the Makefile for hard-coded paths)
-    endif
+		ifndef CUDNN_FRONTEND_PATH
+			ifeq ($(shell [ -d $(HOME)/cudnn-frontend/include ] && echo "exists"), exists)
+				$(info ✓ cuDNN found, will run with flash-attention)
+				CUDNN_FRONTEND_PATH ?= $(HOME)/cudnn-frontend/include
+			else ifeq ($(shell [ -d cudnn-frontend/include ] && echo "exists"), exists)
+				$(info ✓ cuDNN found, will run with flash-attention)
+				CUDNN_FRONTEND_PATH ?= cudnn-frontend/include
+			else
```

Comment:
Instead of the wrapping logic - try the below change and let me know if this works. Place above the "else" statement.

else ifneq ($(CUDNN_FRONTEND_PATH),)
  $(info ✓ cuDNN found, will run with flash-attention)